CMakeLists.txt

```diff
@@ -788,6 +788,8 @@ elseif(IOS)
 	if(EXISTS "${CMAKE_IOS_SDK_ROOT}/System/Library/Frameworks/GameController.framework")
 		set(nativeExtraLibs ${nativeExtraLibs} "-weak_framework GameController")
 	endif()
+	
+	set(nativeExtraLibs ${nativeExtraLibs} iconv)
```

Comment:
Should we just put this as an else for `if (ICONV_LIBRARY)`?  Apparently it's there.

-[Unknown]